src/main.js

```diff
@@ -40,38 +52,68 @@ function getRandomIndex(array) {
-  saveCover.className = 'save-cover-button hidden';
+  randomCoverButton.className = 'random-cover-button hidden';
+  saveCoverButton.className = 'save-cover-button hidden';
   homeButton.className = 'home-button';
-}
+};
 
 function switchSavedView() {
   homeView.className = 'view home-view hidden';
   savedView.className = 'view saved-view';
   formView.className = 'view form-view hidden';
-  randomCover.className = 'random-cover-button hidden';
-  saveCover.className = 'save-cover-button hidden';
+  randomCoverButton.className = 'random-cover-button hidden';
+  saveCoverButton.className = 'save-cover-button hidden';
   homeButton.className = 'home-button';
-}
+};
 
 function returnHome() {
   homeView.className = 'view home-view';
   formView.className = 'view form-view hidden';
-  randomCover.className = 'random-cover-button';
-  saveCover.className = 'save-cover-button';
+  randomCoverButton.className = 'random-cover-button';
+  saveCoverButton.className = 'save-cover-button';
   homeButton.className = 'home-button hidden';
+};
+
+function submitNewRomCom(event) {
+  event.preventDefault();
+  var submittedRomCom = new Cover(coverInput.value, titleInput.value, firstDescriptorInput.value, secondDescriptorInput.value);
+  newCovers.push(submittedRomCom.cover);
+  newTitles.push(submittedRomCom.title);
+  newDescriptors.push(submittedRomCom.tagline1, submittedRomCom.tagline2);
+  addAllRomComs();
+  returnHome();
+  displayNewRomCom();
+};
+
+function displayNewRomCom() {
+  coverImage.src = coverInput.value;
+  coverTitle.innerText = titleInput.value;
+  firstDescriptor.innerText = firstDescriptorInput.value;
+  secondDescriptor.innerText = secondDescriptorInput.value;
 }
 
-function randomRomCom() {
-  coverImage.src = covers[getRandomIndex(covers)];
-  coverTitle.innerText = titles[getRandomIndex(titles)];
-  firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
-  secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
-//  if (firstDescriptor === secondDescriptor) {
-//    return secondDescriptor.innerText = getRandomIndex(descriptors);
-//  };
-  // storeCover();
+function randomRomComReload() {
+  var currentRomCom = new Cover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)]);
+  coverImage.src = currentRomCom.cover;
+  coverTitle.innerText = currentRomCom.title;
+  firstDescriptor.innerText = currentRomCom.tagline1;
+  secondDescriptor.innerText = currentRomCom.tagline2;
+};
+
+function randomRomComButton() {
+  addAllRomComs();
+  var allRomCom = new Cover(allCovers[getRandomIndex(allCovers)], allTitles[getRandomIndex(allTitles)], allDescriptors[getRandomIndex(allDescriptors)], allDescriptors[getRandomIndex(allDescriptors)]);
+  coverImage.src = allRomCom.cover;
+  coverTitle.innerText = allRomCom.title;
+  firstDescriptor.innerText = allRomCom.tagline1;
+  secondDescriptor.innerText = allRomCom.tagline2;
```

Comment:
Great catch with inputting a new object instance of the Cover class here to make the function more dynamic.